yb-voyager/cmd/exportData.go

```diff
@@ -186,6 +185,13 @@ func exportDataOffline() bool {
 			log.Errorf("Export Data using debezium failed: %v", err)
 			return false
 		}
+
+		if changeStreamingIsEnabled(exportType) {
+			log.Infof("live migration complete, proceeding to cutover")
+			triggerName := getTriggerName("", "exporter", source.DBType)
+			createTriggerIfNotExists(triggerName)
```

Comment:
Once https://github.com/yugabyte/yb-voyager/pull/1007 gets merged, we should use metaDB to record this state.